File: Java/program54.c
///////////////////////////////////////////////////////
//
//  File name :     program54.c
//  Descreption :   Accept Input and Perform Addition And Display Result  
//  Author :        Vaibhav
//  Date :          20/05/2025
//
///////////////////////////////////////////////////////

#include<stdio.h>

int main()
{
    int iNo1 = 0,iNo2 = 0;
    int iResult = 0;

    printf("---- This Application is used to perform Addition ----\n");

    printf("Enter First Number: ");
    scanf("%d",&iNo1);

    printf("Enter Second Number: ");
    scanf("%d",&iNo2);

    iResult = iNo1+ iNo2;

    printf("Addition is : %d",iResult);

    return 0;
}

File: program54.java
///////////////////////////////////////////////////////
//
//  File name :     program54.java
//  Descreption :   Accept Input and Perform Addition And Display Result  
//  Author :        Vaibhav
//  Date :          
//
///////////////////////////////////////////////////////


import java.util.*;


class program54
{
    public static void main(String A[])
    {
        Scanner sobj = null;
        int iNo1 = 0,iNo2 = 0, iAns = 0;
        sobj = new Scanner(System.in);
        System.out.println("---- This Application is used to perform Addition ----");
        System.out.println("Enter First number: ");
        iNo1 = sobj.nextInt();

        System.out.println("Enter Second number: ");
        iNo2 = sobj.nextInt();

        iAns = iNo1+iNo2;

        System.out.println("Addition is : "+iAns);
    }
}